main.py

```diff
@@ -0,0 +1,333 @@
+    state['sitting_start_time'] = None
+    state['stretch_prompted'] = False
+    state['consecutive_bad_posture'] = 0
+    state['prev_is_standing'] = False
+    # サウンドループを停止（ここではオーディオオブジェクトは不要）
+    state = manage_sound_loop(state, False, None)
+    return state
+
+# ─────────────────────────────────────────────────────────────
+#  メイン関数
+# ─────────────────────────────────────────────────────────────
+def main():
+    """ メイン処理 """
+    # 初期化
+    # initialize_akari()
+    suppress_logs()
+    pipeline = setup_depthai_pipeline()
+
+    # 状態変数を辞書で管理
+    state = {
+        'bad_posture_count': 0,
+        'consecutive_bad_posture': 0,
+        'last_detection_time': 0.0,
+        'sitting_start_time': None,
+        'stretch_prompted': False,
+        'prev_is_standing': False,
+        'stand_up_time': None,
+        'ping_thread': None,
+        'ping_active': False
+    }
+
+    # ★ 変更: simpleaudio のためにサウンドファイルを事前に読み込む
+    print("🎵  サウンドファイルを読み込んでいます ...")
+    try:
+        siren_long_obj = sa.WaveObject.from_wave_file("sound/siren_long.wav")
+        siren_short_obj = sa.WaveObject.from_wave_file("sound/siren_short.wav")
+        reset_obj = sa.WaveObject.from_wave_file("sound/reset.wav")
+    except FileNotFoundError as e:
+        print(f"エラー: サウンドファイルが見つかりません: {e}", file=sys.stderr)
+        print("'sound' ディレクトリに 'siren_long.wav', 'siren_short.wav', 'reset.wav' が存在することを確認してください。", file=sys.stderr)
+        sys.exit(1)
+
+    print("📷  DepthAI カメラ起動中。'q' で終了")
+    
+    with dai.Device(pipeline) as device, mp_pose.Pose(
+        min_detection_confidence=0.5,
+        min_tracking_confidence=0.5) as pose:
+
+        rgb_queue = device.getOutputQueue("rgb", maxSize=4, blocking=False)
+
+        while True:
+            in_rgb = rgb_queue.tryGet()
+            if in_rgb is None:
+                cv2.waitKey(1)
+                continue
+
+            frame = in_rgb.getCvFrame()
+            if frame is None:
+                continue
+                
+            # MediaPipeでの処理
+            imgRGB = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
+            imgRGB.flags.writeable = False
+            results = pose.process(imgRGB)
+            imgRGB.flags.writeable = True
+            img = cv2.cvtColor(imgRGB, cv2.COLOR_RGB2BGR)
+
+            # 姿勢ランドマークが検出されたかどうかで処理を分岐
+            if results.pose_landmarks:
+                # ★ 変更: 読み込んだオーディオオブジェクトを関数に渡す
+                state = process_landmarks(img, results.pose_landmarks, state, siren_long_obj, siren_short_obj, reset_obj)
+            else:
+                state = handle_no_detection(img, state)
+
+            # タイマーを常時表示
+            h, w, _ = img.shape
+            if state['sitting_start_time']:
+                elapsed = time.time() - state['sitting_start_time']
+                draw_text(img, f"Sitting {hms(elapsed)}", (w-270, h-20), (0,0,0), (255,255,255))
+            else:
+                draw_text(img, "Standing 00:00:00", (w-270, h-20), (0,0,0), (255,255,255))
```

Comment:
```suggestion
                draw_text(img, "Standing 00:00:00", (w-300, h-20), (0,0,0), (255,255,255))
```

Standingの時間表示の末尾が切れています。